packages/fuels/tests/contracts.rs

```diff
@@ -1071,53 +1071,26 @@ async fn test_add_custom_assets() -> Result<()> {
 }
 
 #[tokio::test]
-async fn test_contract_raw_slice() -> Result<()> {
```

Comment:
I might be confused by the git situation, but why are we removing this raw_slice test? I think this was updated (or removed) in the raw slice as input PR, but not sure why this is being touched in this PR 🤔

Reply:
That was a merge issue. My branch reintroduced that test somehow so I had to remove it. 
The raw_slice PR moved the test to `types_contracts.rs` https://github.com/FuelLabs/fuels-rs/blob/f5d08d67543ee35fedc561c5c8335bb134ec24fd/packages/fuels/tests/types_contracts.rs#L1828